helpers/deploymentConfig.ts

```diff
@@ -0,0 +1,174 @@
+import { contracts as governanceBscMainnet } from "@venusprotocol/governance-contracts/deployments/bscmainnet.json";
+import { contracts as governanceBscTestnet } from "@venusprotocol/governance-contracts/deployments/bsctestnet.json";
+import { ethers } from "hardhat";
+import { DeploymentsExtension } from "hardhat-deploy/types";
+
+import { addresses as venusProtocolBscMainnet } from "../deployments/bscmainnet_addresses.json";
+import { addresses as venusProtocolBscTestnet } from "../deployments/bsctestnet_addresses.json";
+import { convertToUnit } from "./utils";
+
+export enum InterestRateModels {
+  WhitePaper,
+  JumpRate,
+}
+
+export type TokenConfig = {
+  isMock: boolean;
+  name?: string;
+  symbol: string;
+  decimals?: number;
+  tokenAddress: string;
+  faucetInitialLiquidity?: boolean;
+};
+
+export type VTokenConfig = {
+  name: string;
+  symbol: string;
+  asset: string; // This should match a name property from a TokenCofig
+  rateModel: string;
+  baseRatePerYear: string;
+  multiplierPerYear: string;
+  jumpMultiplierPerYear: string;
+  kink_: string;
+  collateralFactor: string;
+  liquidationThreshold: string;
+  reserveFactor: string;
+  initialSupply: string;
+  supplyCap: string;
+  borrowCap: string;
+  vTokenReceiver: string;
+};
+
+export type PreconfiguredAddresses = { [contract: string]: string };
+
+export type DeploymentConfig = {
+  tokensConfig: TokenConfig[];
+  marketsConfig: VTokenConfig[];
+  preconfiguredAddresses: PreconfiguredAddresses;
+};
+
+export type NetworkConfig = {
+  hardhat: DeploymentConfig;
+  bsctestnet: DeploymentConfig;
+  bscmainnet: DeploymentConfig;
+};
+
+const preconfiguredAddresses = {
+  hardhat: {
+    VTreasury: "account:deployer",
+  },
+  bsctestnet: {
+    VTreasury: venusProtocolBscTestnet.VTreasury,
+    NormalTimelock: governanceBscTestnet.NormalTimelock.address,
+    VTokenImpl: "0x8d79c8f4400fe68fd17040539fe5e1706c1f2850",
+  },
+  bscmainnet: {
+    VTreasury: venusProtocolBscMainnet.VTreasury,
+    NormalTimelock: governanceBscMainnet.NormalTimelock.address,
+    VTokenImpl: "0xc3279442a5acacf0a2ecb015d1cddbb3e0f3f775",
+  },
+};
+
+export const globalConfig: NetworkConfig = {
+  hardhat: {
+    tokensConfig: [],
```

Comment:
Should we have a similar setup to testnet for hardhat? The idea is that we can use the scripts to easily deploy the protocol for local testing / simulations

Reply:
I believe that all of the previous deployments have been imported. Here is the PR - https://github.com/VenusProtocol/venus-protocol/pull/387

Reply:
VEN-2296 to track this